assets/js/phoenix.js

```diff
@@ -1196,9 +1196,7 @@ export class Ajax {
       let req = new XDomainRequest() // IE8, IE9
```

Comment:
We could technically remove this. Current worldwide usage for both combined is at < 1%.